api/src/main/java/io/minio/messages/EventType.java

```diff
@@ -38,6 +38,12 @@ public enum EventType {
   OBJECT_REMOVED_ANY("s3:ObjectRemoved:*"),
   OBJECT_REMOVED_DELETE("s3:ObjectRemoved:Delete"),
   OBJECT_REMOVED_DELETED_MARKER_CREATED("s3:ObjectRemoved:DeleteMarkerCreated"),
+  OBJECT_REPLICATION_ANY("s3:Replication:*"),
+  OBJECT_REPLICATION_FAILED("s3:Replication:OperationFailedReplication"),
+  OBJECT_REPLICATION_COMPLETE("s3:Replication:OperationCompletedReplication"),
+  OBJECT_REPLICATION_NOT_TRACKED("s3:Replication:OperationNotTracked"),
+  OBJECT_REPLICATION_MISSED_THRESHOLD("s3:Replication:OperationMissedThreshold"),
+  OBJECT_REPLICATION_REPLICATED_AFTER_THRESHOLD("s3:Replication:OperationReplicatedAfterThreshold"),
```

Comment:
```suggestion
  REPLICATION_ANY("s3:Replication:*"),
  REPLICATION_OPERATION_FAILED_REPLICATION("s3:Replication:OperationFailedReplication"),
  REPLICATION_OPERATION_COMPLETED_REPLICATION("s3:Replication:OperationCompletedReplication"),
  REPLICATION_OPERATION_NOT_TRACKED("s3:Replication:OperationNotTracked"),
  REPLICATION_OPERATION_MISSED_THRESHOLD("s3:Replication:OperationMissedThreshold"),
  REPLICATION_OPERATION_REPLICATED_AFTER_THRESHOLD(
      "s3:Replication:OperationReplicatedAfterThreshold"),
```

Reply:
I've used the minio-go code as reference, where they prefixed the Replication with Object, see https://github.com/minio/minio/blob/master/internal/event/name.go

Your call!

Reply:
Object prefix is not correct. We have to stick to the actual name like I mentioned in the comment.